client/app/match/match.js

```diff
@@ -24,8 +17,7 @@ angular.module( 'dinnerDaddy.match', ['dinnerDaddy.services'] )
         $scope.restaurants = data;
         $scope.currRestaurant = $scope.restaurants[currRestaurantIndex];
         var currentImageURL = $scope.currRestaurant.image_url;
```

Comment:
do we need to set this in-between currentImageURL variable? Or is it just for readability on line 20?